### PR TITLE
First pass at document for ImageJ2 tools.

### PR DESCRIPTION
**Pull request by gregvonkuster**
@bgruening Hope this looks ok for now - let me know if you see anything you don't like and I'll update.  I'm planning to keep this current with additional tools.

### CODE REVIEW

File: tools/image_processing/imagej2/readme.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+The Galaxy ImageJ2 tools currently fall into 3 general categories, binary image tools, and BunwarpJ plugin tools and a few "other" tools (e.g., Add or remove noise).  Additional tools will be added at a steady pace, and these miscellaneous other tools will fall into more defined categories.
+
+Galaxy Binary Image Tools
+=========================
+
+These Galaxy tools wrap the ImageJ2 plugins that are available in the ImageJ2 Process -> Binary menu.  The current tool set consists of the following.
```
Comment (bgruening) [inline tools/image_processing/imagej2/readme.md]: You can use: →

File: tools/image_processing/imagej2/readme.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+
+Using Fiji with Galaxy tools
+============================
+
+Galaxy ImageJ2 tool wrappers generate a command line that calls a Python script, passing it a series of arguments including a Jython script named jython_script.py that resides in the same directory as the tool wrapper.  During tool execution, the Python script will call ImageJ2 with the --headless argument to run without the ImageJ2 GUI.  The Jython script is also passed to ImageJ2 along with all command line arguments that it expects.  ImageJ2 will execute the Jython script, passing the expected arguments.  The command line to run ImageJ2 from a Galaxy tool wrapper looks something like this:
+
+`ImageJ2 --ij2 --headless --jython ~jython_script.py arg1, arg2, ...`
+
+Each tool execution starts the ImageJ2 application within a Java virtual machine (JVM).  When ImageJ2 is finished processing the Jython script, the results are either written to a file or returned to the calling Galaxy process.  The JVM is shut down, and the Galaxy job terminates.
```
Comment (bgruening) [inline tools/image_processing/imagej2/readme.md]: I would squeeze HPC somewhere in here. With this in place you can run your tools on any supported Galaxy HPC env.
Also it's probably nice to point out, that we can only support `--headless` IJ plugins.

**Comment by bgruening** [comment]
I like really like this!

**Comment by gregvonkuster** [comment]
@bgruening Thanks for the feedback - I think I have all of it now included in the 2nd commit.

**Comment by bgruening** [comment]
Great! Very nice documentation.

**Pull request merged**